src/bp/nlu-core/language/systemExamples.ts

```diff
@@ -0,0 +1,39 @@
+import fse from 'fs-extra'
+import _ from 'lodash'
+import path from 'path'
+
+const SystemExamplesByLang: _.Dictionary<SystemExamples> = {}
+
+export type SystemExamples = {
```

Comment:
can you only move your type at top of file ?

Reply:
:+1: